packages/default/test/shared.js

```diff
@@ -1574,90 +1640,92 @@ export const sharedTestCases = (mode, torusSP, storageLayer, keyType) => {
 
-      // This test require development API, only work with local/beta env
-      const metadataUrl = getMetadataUrl();
-      // if (metadataUrl === "https://node-1.dev-node.web3auth.io/metadata") metadataUrl = "https://metadata-testing.tor.us";
+    //   // This test require development API, only work with local/beta env
+    //   const metadataUrl = getMetadataUrl();
+    //   // if (metadataUrl === "https://node-1.dev-node.web3auth.io/metadata") metadataUrl = "https://metadata-testing.tor.us";
 
-      await getOrSetNonce(metadataURL, 0, keyType, postboxKeyBN, false);
-      /*
-      // Not available on latest metadata server
-      await post(
-        `${metadataUrl}/set_nonce`,
-        {
-          pub_key_X: pubKeyPoint.x.toString("hex"),
-          pub_key_Y: pubKeyPoint.y.toString("hex"),
-          key_type: keyType,
-        },
-        undefined,
-        { useAPIKey: true }
-      );
-      */
-      // Call get or set nonce
-      /*
-      const serviceProvider = new TorusServiceProvider({
-        postboxKey: postboxKeyBN.toString("hex"),
-        customAuthArgs: {
-          enableOneKey: true,
-          metadataUrl,
-          // This url has no effect as postbox key is passed, passing it just to satisfy direct auth checks.
-          baseUrl: "http://localhost:3000",
-          web3AuthClientId: "test",
-          network: "mainnet",
-        },
-        keyType,
-      });
-      */
+    //   await getOrSetNonce(metadataURL, 0, keyType, postboxKeyBN, false);
+    //   /*
+    //   // Not available on latest metadata server
+    //   await post(
+    //     `${metadataUrl}/set_nonce`,
+    //     {
+    //       pub_key_X: pubKeyPoint.x.toString("hex"),
+    //       pub_key_Y: pubKeyPoint.y.toString("hex"),
+    //       key_type: keyType,
+    //     },
+    //     undefined,
+    //     { useAPIKey: true }
+    //   );
+    //   */
+    //   // Call get or set nonce
+    //   /*
+    //   const serviceProvider = new TorusServiceProvider({
+    //     postboxKey: postboxKeyBN.toString("hex"),
+    //     customAuthArgs: {
+    //       enableOneKey: true,
+    //       metadataUrl,
+    //       // This url has no effect as postbox key is passed, passing it just to satisfy direct auth checks.
+    //       baseUrl: "http://localhost:3000",
+    //       web3AuthClientId: "test",
+    //       network: "mainnet",
+    //     },
+    //     keyType,
+    //   });
+    //   */
 
-      const res = await getOrSetNonce(metadataUrl, 0, keyType, postboxKeyBN);
-      // equal(res.typeOfUser, "v1"); no longer returned
+    //   const res = await getOrSetNonce(metadataUrl, 0, keyType, postboxKeyBN);
+    //   // equal(res.typeOfUser, "v1"); no longer returned
 
-      const anotherRes = await getOrSetNonce(metadataUrl, 0, keyType, postboxKeyBN);
-      deepEqual(res, anotherRes);
-    });
+    //   const anotherRes = await getOrSetNonce(metadataUrl, 0, keyType, postboxKeyBN);
+    //   deepEqual(res, anotherRes);
+    // });
```

Comment:
Why commented out? Remove?

Reply:
this test case was not testing anything on 266,  i think i can re add it with assert statements